File: BAEKJOON/java/21867_Java_Bitecode.java
import java.io.*;

public class Main {
    public static String s;

    public static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

    public static void main(String[] args) throws IOException {
        input();
        solution();

        bw.flush();
        bw.close();
    }

    private static void input() throws IOException {
        int n = Integer.parseInt(br.readLine());
        s = br.readLine();
    }

    private static void solution() throws IOException {
        StringBuilder sb = new StringBuilder();

        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) == 'J' || s.charAt(i) == 'A' || s.charAt(i) == 'V') {
                continue;
            }

            sb.append(s.charAt(i));
        }

        if (sb.toString().isEmpty()) {
            bw.write("nojava");
        } else {
            bw.write(sb.toString());
        }
    }
}

File: BAEKJOON/cpp/21867_Java_Bitecode.cpp
#include <iostream>
#include <string>
using namespace std;

int n;
string s, answer;

void input() {
	cin >> n;
	cin >> s;
}

void solution() { 
	for (int i = 0; i < s.size(); i++) {
		if (s[i] == 'J' || s[i] == 'A' || s[i] == 'V') {
			continue;
		}

		answer += s[i];
	}

	if (answer.empty()) {
		cout << "nojava";
	}
	else {
		cout << answer;
	}
}

int main() {
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	input();
	solution();

	return 0;
}